site/content/docs/5.3/dark-mode.md

```diff
@@ -77,3 +77,561 @@ sitemap_exclude: true
+        <div class="accordion-body">
+          <strong>This is the second item's accordion body.</strong> It is hidden by default, until the collapse
+          plugin adds the appropriate classes that we use to style each element. These classes control the overall
+          appearance, as well as the showing and hiding via CSS transitions. You can modify any of this with custom
+          CSS or overriding our default variables. It's also worth noting that just about any HTML can go within the
+          <code>.accordion-body</code>, though the transition does limit overflow.
+        </div>
+      </div>
+    </div>
+  </div>
+</div>
+
+### Dark theme on component
+
+<div class="border border-tertiary p-3">
+  <div class="accordion" id="accordionExample6" data-bs-theme="dark">
+    <div class="accordion-item">
+      <h2 class="accordion-header">
+        <button class="accordion-button" type="button" data-bs-toggle="collapse" data-bs-target="#collapseOne6"
+                aria-expanded="true" aria-controls="collapseOne6">
+          Accordion Item #1
+        </button>
+      </h2>
+      <div id="collapseOne6" class="accordion-collapse collapse show" data-bs-parent="#accordionExample6">
+        <div class="accordion-body">
+          <strong>This is the first item's accordion body.</strong> It is shown by default, until the collapse plugin
+          adds the appropriate classes that we use to style each element. These classes control the overall
+          appearance, as well as the showing and hiding via CSS transitions. You can modify any of this with custom
+          CSS or overriding our default variables. It's also worth noting that just about any HTML can go within the
+          <code>.accordion-body</code>, though the transition does limit overflow.
+        </div>
+      </div>
+    </div>
+    <div class="accordion-item">
+      <h2 class="accordion-header">
+        <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse"
+                data-bs-target="#collapseTwo6" aria-expanded="false" aria-controls="collapseTwo6">
+          Accordion Item #2
+        </button>
+      </h2>
+      <div id="collapseTwo6" class="accordion-collapse collapse" data-bs-parent="#accordionExample6">
+        <div class="accordion-body">
+          <strong>This is the second item's accordion body.</strong> It is hidden by default, until the collapse
+          plugin adds the appropriate classes that we use to style each element. These classes control the overall
+          appearance, as well as the showing and hiding via CSS transitions. You can modify any of this with custom
+          CSS or overriding our default variables. It's also worth noting that just about any HTML can go within the
+          <code>.accordion-body</code>, though the transition does limit overflow.
+        </div>
+      </div>
+    </div>
+  </div>
+</div>
+
+### Light theme on component
+
+<div class="border border-tertiary p-3">
```

Comment:
Don't we need to force the background color here? Or do you prefer to keep it with the background of the page to really see if the components have a bg themselves?

```suggestion
<div class="border border-tertiary p-3" style="background-color: var(--bs-white)">
```

Same thing to do for all components if accepted